### DISCUSSION TITLE
Welcome to firefox-ios Discussions!

### DISCUSSION THREAD

**Discussion by dnarcese** [mozilla-mobile/firefox-ios#7709] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Vietthang91** [comment]
> 
> 
> ## 👋 Welcome!
> 
>   We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
>   * Ask questions you’re wondering about.
> 
>   * Share ideas.
> 
>   * Engage with other community members.
> 
>   * Welcome others and are open-minded. Remember that this is a community we
> 
>   build together 💪.
> 
> 

.

**Comment by Bodarobi** [comment]
communityaccounts@apple.com
